modules/infinite-scroll.php

```diff
@@ -63,17 +63,6 @@ private function __construct() {
 	 */
 	public function action_jetpack_modules_loaded() {
 		Jetpack::enable_module_configurable( __FILE__ );
-		add_filter( 'jetpack_module_configuration_url_infinite-scroll', array( $this, 'infinite_scroll_configuration_url' ) );
```

Comment:
The settings under Settings > Reading have an additional option ("Use Google Analytics with Infinite Scroll") that is not found in the Jetpack dashboard settings. Should we keep the old link around until the Jetpack dashboard settings are updated to include all the Infinite Scroll settings?

Reply:
No. I think we should send them to the Jetpack settings view and add the setting for it in another PR.